dfusion_rust_core/src/models/solution.rs

```diff
@@ -17,7 +17,7 @@ pub struct Solution {
 impl Solution {
     pub fn trivial(num_orders: usize) -> Self {
         Solution {
-            prices: vec![1; TOKENS as usize],
+            prices: vec![0; TOKENS as usize],
```

Comment:
Funny, I was under the impression that this was changed a while back...

Reply:
It was, I unfixed it and then re-fixed it